pkg/networkmanager/wireguard.jsx

```diff
@@ -324,6 +348,22 @@ export function WireGuardDialog({ settings, connection, dev }) {
                                     id={idPrefix + '-allowedips-peer-' + i}
                                 />
                             </FormGroup>
+                            <FormGroup className='pf-m-6-col-on-md' label={_("Preshared Key")} fieldId={idPrefix + '-presharedkey-peer-' + i} labelIcon={
+                                <Popover
+                                    bodyContent={_("Preshared key should be a random 32 byte bas64 encoded.")}
+                                >
+                                    <Button variant='link' isInline><HelpIcon /></Button>
+                                </Popover>
+                            }>
+                                <TextInput
+                                    value={peer.presharedKey}
+                                    onChange={(_, val) => {
+                                        setPeers(peers => peers.map((peer, index) => i === index ? { ...peer, presharedKey: val } : peer));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19521-20231024-110617-ce68b93e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)